.travis.yml

```diff
@@ -14,9 +14,9 @@ cache:
   directories:
     - /home/travis/.cargo
 before_script:
-  - rustup component add clippy-preview
+  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then rustup component add clippy-preview; fi
```

Comment:
I should probably reverse this logic to `f [ "$TRAVIS_RUST_VERSION" != "beta" ]; then`